src/map.h

```diff
@@ -1212,6 +1212,10 @@ void add_corpse( const tripoint &p );
          * Radiation-related plant (and fungus?) death.
          */
         void rad_scorch( const tripoint &p, int time_since_last_actualize );
+        /**
+         * Decays cosmetic fields.
+         */
+        void decay_gibs( const tripoint &p, int time_since_last_actualize );
```

Comment:
`decay_cosmetic_fields()` is much better. The name is clearer, no pun, no need to comment.